packages/gatsby-theme-newrelic/src/components/SimpleFeedback.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { graphql, useStaticQuery } from 'gatsby';
+import { css } from '@emotion/core';
+
+import Button from './Button';
+import Icon from './Icon';
+import PageTools from './PageTools';
+
+const SimpleFeedback = ({ title, slug, labels }) => {
+  const { site } = useStaticQuery(graphql`
+    query FeedbackQuery {
+      site {
+        siteMetadata {
+          siteUrl
+          repository
+        }
+      }
+    }
+  `);
+
+  const { repository, siteUrl } = site.siteMetadata;
+  const issueUrl = `${repository}/issues/new`;
+
+  const issueTitle = title
+    ? `Feedback:+${title.replace(' ', '+')}`
+    : 'Website Feedback';
+
+  const body =
+    title && slug ? `&body=Page:%20[${title}](${siteUrl}${slug})` : '';
+
+  const positiveFeedback = `${issueUrl}?labels=${labels.join(
+    ','
+  )},feedback-positive&title=${issueTitle}${body}`;
+  const negativeFeedback = `${issueUrl}?labels=${labels.join(
+    ','
+  )},feedback-negative&title=${issueTitle}${body}`;
+
+  return (
+    <PageTools.Section>
+      <PageTools.Title>Feedback</PageTools.Title>
+      <div
+        css={css`
+          font-size: 0.875rem;
+        `}
+      >
+        Was this page helpful?
+      </div>
+      <div
+        css={css`
+          display: flex;
+          margin-top: 0.5rem;
+          justify-content: center;
+          align-items: flex-start;
+
+          a {
+            flex-grow: 1;
+          }
+
+          > *:first-child {
```

Comment:
[nit] You can totally leverage `gap` if you want to use it in browsers that support it! You just have to use the [`@supports` at-rule](https://developer.mozilla.org/en-US/docs/Web/CSS/@supports).

```js
css`
  gap: 0.5rem;

  @supports not (gap: 0.5rem) {
    > *:first-child {
      margin-right: 0.5rem;
    }
  }
`
```